mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -1,52 +1,63 @@
-import {Navigate, useParams} from "react-router-dom";
+import {useEffect, useState} from "react";
 import {HeadingLevel, Title} from "src/component/title/Title";
-import {QueryParamTypes} from "src/logic/QueryParamTypes";
+import {UserPreviewDAL} from "src/dataAccessLogic/UserPreviewDAL";
 import {FavoriteWaysTable} from "src/logic/waysTable/FavoriteWaysTable";
 import {MentoringWaysTable} from "src/logic/waysTable/MentoringWaysTable";
 import {OwnWaysTable} from "src/logic/waysTable/OwnWaysTable";
-import {pages} from "src/router/pages";
 import styles from "src/logic/userPage/UserPage.module.scss";
 
+/**
+ * PageProps
+ */
+interface UserPageProps {
+
+  /**
+   * Page's uuid
+   */
+  uuid: string;
+}
+
 /**
  * User page
  */
-export const UserPage = () => {
-  const {uuid} = useParams<QueryParamTypes>();
+export const UserPage = (props: UserPageProps) => {
+  const [userName, setUserName] = useState<string>("");
+
+  /**
+   * Load user
+   */
+  const getUser = async () => {
+    const user = await UserPreviewDAL.getUserPreview(props.uuid);
+    setUserName(user.name);
```

Comment:
Please do something if user is not found

Reply:
We created ValidateParams wrapper for check params (uuid and others). Or maybe I don't understand you?